From aa9ad483f2eb5337c844336db2379e659cb5853e Mon Sep 17 00:00:00 2001
From: ccobine <carol.cobine@joyent.com>
Date: Fri, 15 Mar 2019 15:39:16 -0400
Subject: [PATCH] DOC-648 Please add that MD5 headers require base64 when
 adding a header for upload (fix markdown syntax) Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 docs/user-guide/storage-reference.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/user-guide/storage-reference.md b/docs/user-guide/storage-reference.md
index 1691c40..f7124f6 100644
--- a/docs/user-guide/storage-reference.md
+++ b/docs/user-guide/storage-reference.md
@@ -1,4 +1,4 @@
- ---
+---
 title: Object Storage Reference
 markdown2extras: wiki-tables, code-friendly
 ---
-- 
2.21.0

